sdk/identity/Azure.Identity/src/VisualStudioCodeCredential.cs

```diff
@@ -38,7 +38,8 @@ public class VisualStudioCodeCredential : TokenCredential
         /// <param name="options">Options for configuring the credential.</param>
         public VisualStudioCodeCredential(VisualStudioCodeCredentialOptions options) : this(options, default, default, default, default) { }
 
-        internal VisualStudioCodeCredential(VisualStudioCodeCredentialOptions options, CredentialPipeline pipeline, MsalPublicClient client, IFileSystemService fileSystem, IVisualStudioCodeAdapter vscAdapter)
+        internal VisualStudioCodeCredential(VisualStudioCodeCredentialOptions options, CredentialPipeline pipeline, MsalPublicClient client, IFileSystemService fileSystem,
+            IVisualStudioCodeAdapter vscAdapter)
```

Comment:
It looks like this file only contains formatting changes. Did you mean to include in this PR?

Reply:
Nope - I've been experimenting with IDE settings, and the format was doing unexpected things :)